src/app/services/time-tracking/time.table.ts

```diff
@@ -48,6 +48,7 @@ export class TimeTable implements DatabaseTable<TimeEntry> {
                     ends: timeEntries.map(entry => entry.end),
                     utcDate: timeEntries[0].utcDate,
                     duration: timeEntries.reduce((duration, current) => duration.plus(current.duration), Duration.fromMillis(0)),
+                    isNonWorkday: timeEntries.some(entry => entry.isNonWorkday),
```

Comment:
Shouldn't it be `every` here instead of `some`? If data gets mixed, it may be better to treat it as a working day?